core/services/request/sanitizers/MultipleAttributes.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace EventEspresso\core\services\request\sanitizers;
+
+/**
+ * Class MultipleAttributes
+ *
+ * @author  Hossein Rafiei
+ * @package EventEspresso\core\services\request\sanitizers
+ * @since   4.10.30
```

Comment:
using the following will swap the placeholder for whatever version this actually gets released under

```php
 * @since   $VID:$
```

Reply:
Nice.
What will change that to the latest version? Our own script or Github itself?

Reply:
currently it's a script on the `buildmachine` server that packages things up for release.
there has been some work done to migrate that to GitHub actions, but it's not fully completed yet.
eventually everything will be done via GitHub actions as part of our CI/CD pipeline.

Reply:
OH and FYI, that tag originally came from phpStorm and you can have that added automatically by editing your live templates:

<img width="919" alt="Screenshot 2022-05-26 102917" src="https://user-images.githubusercontent.com/1751030/170542876-ba48b6d0-c508-457b-a271-16131d7e1540.png">